libs/wingsdk/test/target-sim/logger.test.ts

```diff
@@ -33,3 +33,23 @@ test("inflight uses a logger", async () => {
   expect(listMessages(s)).toMatchSnapshot();
   expect(app.snapshot()).toMatchSnapshot();
 });
+
+test("Logger has display hidden property set to true", async () => {
```

Comment:
Could we also add a test somewhere that built-in resources (Bucket, Counter, Function) are not hidden by default?